monai/data/image_reader.py

```diff
@@ -131,9 +131,10 @@ def _stack_images(image_list: List, meta_dict: Dict):
     if len(image_list) <= 1:
         return image_list[0]
     if meta_dict.get("original_channel_dim", None) not in ("no_channel", None):
-        raise RuntimeError("can not read a list of images which already have channel dimension.")
+        channel_dim = int(meta_dict["original_channel_dim"])
+        return np.concatenate(image_list, axis=channel_dim)
     meta_dict["original_channel_dim"] = 0
-    return np.stack(image_list, axis=0)
+    return np.stack(image_list, axis=meta_dict["original_channel_dim"])
```

Comment:
Hi @wyli ,

I think we don't need to change this line, as `meta_dict["original_channel_dim"]` just set to `0` in above line, the axis is always `0` here.

Thanks.